src/SFA.DAS.EmployerIncentives.Queries/ApprenticeshipIncentives/ApprenticeshipIncentiveHasPossibleChangeOrCircs/ApprenticeshipIncentiveHasPossibleChangeOrCircsRequest.cs

```diff
@@ -1,15 +1,28 @@
 ﻿using System;
+using SFA.DAS.EmployerIncentives.Abstractions.Logging;
 using SFA.DAS.EmployerIncentives.Abstractions.Queries;
 
 namespace SFA.DAS.EmployerIncentives.Queries.ApprenticeshipIncentives.ApprenticeshipIncentiveHasPossibleChangeOrCircs
 {
-    public class ApprenticeshipIncentiveHasPossibleChangeOrCircsRequest : IQuery
+    public class ApprenticeshipIncentiveHasPossibleChangeOrCircsRequest : IQuery, ILogWriter
```

Comment:
Also add the ILogWriter interface to the reponse object ApprenticeshipIncentiveHasPossibleChangeOrCircsResponse then we can log the value returned form the query in the decorator so the original logging that was in the orchestrator is retained.  Also do for some of the other  logging that was in the orchestrator activities.  Then we can remove al the logging from the Activities to clean up the code and maintain good separation of concerns while still maintaining the original functionality.